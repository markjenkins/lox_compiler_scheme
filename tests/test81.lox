// from craftinginterpreters/test/block/scope.lox

var a = "outer";

{
  var a = "inner";
  print a; // expect: inner
}

print a; // expect: outer
