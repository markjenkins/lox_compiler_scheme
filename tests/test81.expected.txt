OP_CONSTANT "outer"
OP_DEFINE_GLOBAL "a"
OP_CONSTANT "inner"
OP_GET_LOCAL 1
OP_PRINT
OP_POP
OP_GET_GLOBAL "a"
OP_PRINT
OP_RETURN
